airbyte-cdk/python/airbyte_cdk/sources/declarative/yaml_declarative_source.py

```diff
@@ -14,6 +14,7 @@ class YamlDeclarativeSource(DeclarativeSource):
     def __init__(self, path_to_yaml):
         self._factory = DeclarativeComponentFactory()
         self._source_config = self._read_and_parse_yaml_file(path_to_yaml)
+        self.logger = logging.getLogger(f"airbyte.{self.name or ''}")
```

Comment:
what would cause `self.name` to be None?

Reply:
Probably me just being overly cautious, will remove.